dev3/2024-07-02/example-metadata/min-specimen-biosample.json

```diff
@@ -0,0 +1,74 @@
+{
+    "@context": [
+        "https://w3id.org/ro/crate/1.1/context",
+        {
+            "organism_classification": "https://schema.org/taxonomicRange",
+            "BioChemEntity": "https://schema.org/BioChemEntity",
+            "obo": "http://purl.obolibrary.org/obo/",
+            "acquisiton_method": {
+                "@reverse": "https://schema.org/result",
```

Comment:
TIL `@reverse`. It's interesting that I've not run into it before. Thanks, @sherwoodf. It does leave me wondering whether or not that bumps us to a higher-level of complexity.

Reply:
As you'll see i mention a lot below, this was the result of me a) not finding very specific ontology terms that i think would fit better and b) trying to preserve what felt like a more sensible sequence when reading the json objects. Ideally we would have better terms to connect these objects.

Reply:
- a) definitely understood. Under the GIDE banner, I assume we can also consider either updating or creating ontologies as necessary. I don't assume we want to do that for the challenge, so maybe more a question of just documenting whether use of idioms like `@reverse` are temporary or not.
 - b) Big :thumbsup: for a focus on readability.